plugins/skylog/example_skylog_test.go

```diff
@@ -0,0 +1,61 @@
+//
```

Comment:
I think the test code should be put in [test/e2e/cases](https://github.com/SkyAPM/go2sky/tree/master/test/e2e/cases)

Reply:
fixed